CHANGES.rst

```diff
@@ -20,11 +20,16 @@ Unreleased
     (`#3059`_)
 -   :func:`send_file` supports :class:`~io.BytesIO` partial content.
     (`#2957`_)
+-   Use Werkzeug's JSONMixin and delete Flask's JSONMixin class;
+    Bump minimum dependency versions to the latest stable versions:
+    Werkzeug >= 0.15
```

Comment:
Move the Werkzeug >= 0.15 part to its own entry at the top of the changelog so it's more visible.